cohortextractor/cohortextractor.py

```diff
@@ -371,6 +442,12 @@ def main():
         type=str,
         default=os.environ.get("TEMP_DATABASE_NAME", ""),
     )
+    generate_cohort_parser.add_argument(
+        "--index-date-range",
```

Comment:
Thinking aloud: we've talked about making `index-date` a compulsory argument, which I like & think we should implement soon. When we do, would this be a mutually-exclusive option group of either `--index-date` OR `--index-date-range`?  I think that sounds right.

Reply:
I'm not 100% sure about that. I feel like, for a normal study (as opposed to a measure definition) the index date belongs with the rest of the study definition rather than as something supplied on the command line.

I think we would want to encourage people to make use of it, rather than hardcoding dates throughout the study, but that's a slightly different issue.

Reply:
I agree